pkg/operator/defaults.go

```diff
@@ -103,5 +103,6 @@ var (
 	// This is only added for users to try the unstable versions.
 	PrometheusExperimentalVersions = []string{
 		"v3.0.0-beta.0",
+		"v2.55.0-rc.0",
```

Comment:
As we discussed in https://github.com/prometheus-operator/prometheus-operator/issues/7002#issuecomment-2406740176, option 2 is what we can do for the fix

Reply:
Ok, then should the change be done here https://github.com/prometheus-operator/prometheus-operator/blob/main/pkg/prometheus/promcfg.go#L112

Reply:
here https://github.com/prometheus-operator/prometheus-operator/blob/d1fb4196594218b9aea80bfb588881c293fb6e56/pkg/prometheus/promcfg.go#L4268-L4269